src/crypto/cn/CryptoNight_x86.h

```diff
@@ -1375,7 +1376,10 @@ void cryptonight_quad_hash_zen(const uint8_t* __restrict__ input, size_t size, u
     uint64_t idx2 = al2;
     uint64_t idx3 = al3;
 
-    __m128i cx0, cx1, cx2, cx3;
+    __m128i cx0 = _mm_setzero_si128();
+    __m128i cx1 = _mm_setzero_si128();
+    __m128i cx2 = _mm_setzero_si128();
+    __m128i cx3 = _mm_setzero_si128();
```

Comment:
Same. Even though it's not the main loop, it's not needed. Probably a false positive.